modules/destroy-hc-ibm-z-cli.adoc

```diff
@@ -17,18 +17,7 @@ To destroy a hosted cluster and its managed cluster on `x86` bare metal with {ib
 $ oc -n <hosted_cluster_namespace> scale nodepool <nodepool_name> --replicas 0
 ----
 +
-After the `NodePool` object is scaled to `0`, the compute nodes are detached from the hosted cluster. In {product-title} version 4.17, this function is applicable only for {ibm-z-title} with KVM. For z/VM and LPAR, you must delete the compute nodes manually.
-+
-If you want to re-attach compute nodes to the cluster, you can scale up the `NodePool` object with the number of compute nodes that you want. For z/VM and LPAR to reuse the agents, you must re-create them by using the `Discovery` image.
-+
-[IMPORTANT]
-====
-If the compute nodes are not detached from the hosted cluster or are stuck in the `Notready` state, delete the compute nodes manually by running the following command:
-[source,terminal]
-----
-$ oc --kubeconfig <hosted_cluster_name>.kubeconfig delete node <compute_node_name>
-----
-====
+If you are using  OSA network device in Processor Resource/Systems Manager (PR/SM) mode, auto scaling is not supported. You must delete the old agent manually and scale up the node pool because the new agent joins during the scale down process. 
```

Comment:
```suggestion
If you are using an OSA network device in Processor Resource/Systems Manager (PR/SM) mode, auto scaling is not supported. You must delete the old agent manually and scale up the node pool because the new agent joins during the scale down process. 
```